notebooks/00_quick_start/xdeepfm_single_node.ipynb

```diff
@@ -0,0 +1,405 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# xDeepFM Single Node \n",
+    "This notebook will give you a quick example of how to train an xDeepFM model. \n",
+    "xDeepFM \\[1\\] is a deep learning-based model aims at capturing both lower- and higher-order feature interactions for precise recommender systems. It has the following two key properties:\n",
+    "* It contains a component, named CIN, that learns feature interactions in an explicit fashion and in vector-wise level;\n",
+    "* It contains the tradicition DNN component that learns feature interactions in an implicit fashion and in bit-wise level.\n",
+    "Thus the xDeepFM can learn feature interactions more effectively and manual feature engineering effort can be substantially reduced.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Global Settings and Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append(\"../../\")\n",
+    "from reco_utils.recommender.deeprec.deeprec_utils import *\n",
+    "from reco_utils.recommender.deeprec.models.xDeepFM import *\n",
+    "from reco_utils.recommender.deeprec.IO.iterator import *\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download data\n",
+    "xDeepFM use the FFM format as data input: `<label> <field_id>:<feature_id>:<feature_value>`  \n",
+    "Each line represents an instance, \\<label\\> is a binary value with 1 meaning positive instance and 0 meaning negative instance. \n",
+    "Features are devided into fields. For example, user's gender is a field, it contains three possible values, i.e. male, female and unknown. Occupation can be another fields, which contains many more possible values than the gender field. Both field idx and feature idx are starting from 1. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
```

Comment:
We need to add parameters so this file can be run with papermill. This is very easy and I can do it myself or we can have a quick chat and I will tell you how to do it

Reply:
OK.  I added parameters to this file.